### DISCUSSION TITLE
Display list of contributors

### DISCUSSION THREAD

**Discussion by carlosmarioroldan** [vuejs/vitepress#2212] [Q&A]
Does anyone know if VitePress supports displaying a list of contributors for each page based on Git metadata? I'm currently using VitePress for my documentation site, and I'd like to automatically generate a list of contributors for each page based on the Git revisions for that file. In VuePress, there is a plugin called [vuepress-plugin-contributors](https://github.com/kidonng/vuepress-plugin-contributors) that can do this, but I'm not sure if there is an equivalent plugin or method for VitePress. Any help or suggestions would be greatly appreciated. Thanks!

**Comment by Pezhvak** [comment]
Any updates on this?

**Reply by KomoriDev** [reply]
maybe you can find answer [here](https://nolebase-integrations.ayaka.io/pages/en/), it has a tool that generates changelogs and contributor lists,

**Reply by Pezhvak** [reply]
Great repo, thank you, I'm gonna give it a try 👍🏻

**Comment by brc-dd** [comment]
https://github.com/vuejs/vitepress/issues/3001#issuecomment-1732237283
